cli/dcoscli/main.py

```diff
@@ -67,17 +64,31 @@ def _main():
             msg = 'Unable to get the cluster_id of the cluster.'
             logger.exception(msg)
 
+    # send args call to segment.io
+    reporting_executor = ThreadPoolExecutor(max_workers=2)
+    report = config.get('core.reporting', True)
+    if report and dcoscli.version != 'SNAPSHOT':
```

Comment:
I see that we do this in a couple of places. I think we should move this into the `analytics` module.